### PR TITLE
Fix coverage profile and numbers for CI go_test

### PR DESCRIPTION
**Pull request by dimroc**
## TL;DR
Before, if the coverage didn't come directly from the corresponding test package, it didn't count. Now it does. The change as a result of this PR is illustrated below where we don't have a `logger_test.go` but we do have coverage from other `*_test.go` packages.

image

---

## Details
Our coverage numbers are off with the switch to go1.10 coverprofile. `packageB_test` will not contribute to `packageA` coverage, only `packageA_test` will. This is shown here:

image 6

Where `AddAdapter` clearly has coverage from https://github.com/smartcontractkit/chainlink/blob/10b14d18f20932be8efa25013e7cf6e66355b485/cmd/remote_client_test.go#L228

The answer involves using `-coverpkg=<listOfPackages>` but that then results in erroneous coverage percentages. More information https://github.com/haya14busa/goverage/issues/9#issuecomment-407760156

### CODE REVIEW

**Comment by dimroc** [comment]
TL;DR
Before, if the coverage didn't come directly from the corresponding test package, it didn't count. Now it does. Clearly illustrated below where we don't have a `logger_test.go` but we do have coverage from other `*_test.go` packages.

image
